dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/persistence/file/advanced/FeatureFileOrchestrator.kt

```diff
@@ -64,6 +64,10 @@ internal class FeatureFileOrchestrator(
     )
 
     companion object {
+        private const val BASE_DIR_NAME_REG_EX = "([a-z]+[-|_])+"
```

Comment:
Do we have a list of base dire name used? It seems this regex is allowing a lot more than it should (e.g.: `abcdef|`)